<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryEditor</class>
 <widget class="QWidget" name="InventoryEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="addItem">
        <property name="text">
         <string>Add item</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="deleteItem">
        <property name="text">
         <string>Delete item</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2" rowspan="5">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="3">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Battlesaddle</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="equippedBattlesaddle"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Armour</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QComboBox" name="equippedArmour"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Current ammunition</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Ammo count</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="itemQuantity">
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Quantity</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="currentAmmunition"/>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="ammoCount"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" rowspan="6" colspan="2">
       <widget class="QListWidget" name="itemList"/>
      </item>
      <item row="3" column="3">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Equipped right</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QComboBox" name="equippedRight"/>
         </item>
         <item>
          <widget class="QComboBox" name="equippedRightMode">
           <item>
            <property name="text">
             <string>Mouth</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Magic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Battlesaddle</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3" rowspan="3">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Equipped left</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QComboBox" name="equippedLeft"/>
         </item>
         <item>
          <widget class="QComboBox" name="equippedLeftMode">
           <item>
            <property name="text">
             <string>Mouth</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Magic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Battlesaddle</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
